notebooks/data-sources/TestGrid/metrics/time_to_fail.ipynb

```diff
@@ -0,0 +1,1180 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #50.            expected_frequency = np.array(expected_frequency) * size
</pre><p>I'm a little confused as to why you are scaling the expected frequency array by the length of the input data <code>size</code>.   Why does this work? Also, any reason why your not using the <code><strong>scipy.stats.chisquare</strong></code><strong>  </strong>method here instead of writing your own?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/337/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Thanks for the suggestion. No special reason, we will try out the <code>scipy.stats.chisquare</code>  instead :)</p>

Reply:
We have updated the notebook to use the `scipy.stats.chisquare` library instead. We are scaling the expected frequency so that it is equal in length to the observed frequencies while performing the Chi Square test.